src/cript/nodes/primary_nodes/material.py

```diff
@@ -46,21 +46,13 @@ class Material(PrimaryBaseNode):
         Material names Must be unique within a [Project](../project)
 
     ```json
-    {
-        "name": "my unique material",
-        "component_count": 0,
-        "computational_forcefield_count": 0,
-        "created_at": "2023-03-14T00:45:02.196297Z",
-        "identifier_count": 0,
-        "identifiers": [],
-        "model_version": "1.0.0",
-        "node": "Material",
-        "notes": "",
-        "property_count": 0,
-        "uid": "0x24a08",
-        "updated_at": "2023-03-14T00:45:02.196276Z",
-        "uuid": "403fa02c-9a84-4f9e-903c-35e535151b08",
-    }
+      {
+         "node":["Material"],
+         "name":"my unique material name",
+         "uid":"_:9679ff12-f9b4-41f4-be95-080b78fa71fd",
+         "uuid":"9679ff12-f9b4-41f4-be95-080b78fa71fd"
+         "bigsmiles":"123456",
```

Comment:
for an example, we should use a valid bigsmiles

Reply:
what would I put for this material? I could put the bigsmiles for polystyrene, but not sure if that makes sense or if there is something better.

Reply:
also feel free to make updates to this branch as you see fit, I just put something there for now to build on as we go

Reply:
yeah, polystyrene should be good enough.

We just shouldn't advertise, that we allow invalid bigsmiles strings here.

Reply:
that is very true, I'll update it now!